buildpipeline/DotNet-CoreClr-Trusted-Linux-Crossbuild.json

```diff
@@ -193,7 +193,25 @@
       },
       "inputs": {
         "filename": "docker",
-        "arguments": "run --rm $(DockerCommonRunArgs) ./publish-packages.sh -AzureAccount=$(CloudDropAccountName) -AzureToken=$(CloudDropAccessToken) -BuildType=$(PB_BuildType)  -BuildArch=$(Architecture) -Container=$(Label) -distroRid=$(Rid) -- /p:OverwriteOnPublish=true",
+        "arguments": "run --rm $(DockerCommonRunArgs) ./publish-packages.sh -AzureAccount=$(CloudDropAccountName) -AzureToken=$(CloudDropAccessToken) -BuildType=$(PB_BuildType)  -BuildArch=$(Architecture) -Container=$(Label) -distroRid=$(Rid) -PublishPackages -- /p:OverwriteOnPublish=true",
```

Comment:
I know this was already here but we need to have an issue to remove `/p:OverwriteOnPublish=true` as we shouldn't be allowing overwrite by default. I'm pretty sure that will cause a issue with the identity packages but we should do what corefx did and make sure we only build/publish those from one leg like the windows leg.

Reply:
@wtgodbe please file another issue for this if you haven't already.

Reply:
https://github.com/dotnet/coreclr/issues/14098